lib/src/components/NxFilterInput/NxFilterInput.tsx

```diff
@@ -32,13 +32,14 @@ const NxFilterInput = forwardRef<HTMLDivElement, Props>(
           <div className="nx-text-input__box">
             <NxFontAwesomeIcon icon={faFilter} className="nx-icon--filter-icons" />
             <input type="text"
-                   autoComplete="off"
+                   autoComplete="on"
```

Comment:
Does this screw up non-datalist `NxFilterInput`s?  There's probably a reason it was set to off before

Reply:
> Does this screw up non-datalist `NxFilterInput`s? There's probably a reason it was set to off before

I don't know, but having it off doesn't screw up `datalist` so I reverted it: https://github.com/sonatype/sonatype-react-shared-components/pull/166/commits/f2cdf96869e0c60c6e67ab099f2eec7d930fac50